### PR TITLE
mime type search, color space search fixed

### PR DESCRIPTION
**Pull request by titor-oopart**

### CODE REVIEW

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 1]
```diff
@@ -25,31 +25,31 @@
...
 
     /**
      * This method read all metadata.
-     *
-     * @return
+     * @return void
      */
     public void readAll() {
         String metadata = null;
         list = new ArrayList<>();
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:44]: Define the type of content that contain the new ArrayList<String>();

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 2]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
-     *
-     * @return
+     * @return void
      */
     public static void mimeType(String mimeType) {
-        String imageMimeType = mimeType.substring(0,9);
-        if ((imageMimeType.contains("MIME Type"))) {
-            if (mimeType.contains("image")) {
-                searchMimeType = "image";
+        String mimetype2 = mimeType;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: use descriptive variable name for mimetype2 such as mimetypeAuxiliar or auxMimetype

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 3]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
+        String audioMimeType = mimeType.substring(0, 4);
+        if ((audioMimeType.contains("MIME"))) {
+            if (mimetype2.contains("image")) {
+                int initIndex = mimetype2.indexOf(":");
+                int freeSpace = 2;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: try to define a constant for freeSpace variable

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 4]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
 
     /**
      * This method returns metadata image height.
+     * @return void
      */
     public static void height(String height) {
         if ((height.contains("Image Height"))) {
             int initIndex = height.indexOf(":");
             int endIndex = height.length();
             int freeSpace = 2;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: try to define a constant for freeSpace variable

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 5]
```diff
@@ -101,47 +103,74 @@ public static void width(String width) {
...
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: try to define a constant for freeSpace variable

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 6]
```diff
@@ -25,31 +25,31 @@
...
+        searchColorSpace = null;
         extractMetadata = Runtime.getRuntime().exec(path);
         readAll();
     }
 
     /**
      * This method read all metadata.
-     *
-     * @return
+     * @return void
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: void methods don't have return documentation

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 7]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
 
     /**
      * This method read mime type.
-     *
-     * @return
+     * @return void
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: void methods don't have return documentation.

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 8]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
+                searchMimeType = mimeType;
+            } else {
+                searchMimeType = "All";
             }
         }
     }
 
     /**
      * This method returns metadata image height.
+     * @return void
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: void methods don't have return documentation.

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 9]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
-    /**
-     * This method returns height image.
-     */
-    public static String getHeight() {
-        return searchHeight;
-    }
-
     /**
      * This method returns metadata image Width.
+     * @return void
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: void methods don't have return documentation.

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 10]
```diff
@@ -101,47 +103,74 @@ public static void width(String width) {
...
-    /**
-     * This method returns width image.
-     */
-    public static String getWidth() {
-        return searchWidth;
-    }
-
     /**
      * This method returns metadata image color space.
+     * @return void
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: void methods don't have return documentation.

File: src/main/java/com/fundation/logic/controller/Controller.java [hunk 11]
```diff
@@ -256,6 +256,11 @@ private Criteria getImageCriteria() {
         if (height.length() == 0) {
             height = "0";
         }
+
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/controller/Controller.java]: delete white line

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 12]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
+                searchMimeType = "All";
             }
         }
     }
 
     /**
      * This method returns metadata image height.
+     * @return void
      */
     public static void height(String height) {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: Use descriptive method names as it is difficult to understand what the method does just for the name

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 13]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
 
     /**
      * This method read mime type.
-     *
-     * @return
+     * @return void
      */
     public static void mimeType(String mimeType) {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: Use descriptive method names as it is difficult to understand what the method does just for the name

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 14]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
-     */
-    public static String getHeight() {
-        return searchHeight;
-    }
-
     /**
      * This method returns metadata image Width.
+     * @return void
      */
     public static void width(String width) {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: Use descriptive method names as it is difficult to understand what the method does just for the name

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java (hunk 3 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: done

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java (hunk 4 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/common/MetadataImageExtractor.java]: done

**Pull request merged**